email_campaigns/thanks-contributor-email.md

```diff
@@ -0,0 +1,37 @@
+{{ name }},
+
+# Thanks for attending the Mentored Sprint for Diverse Beginners at PyCon US 2019!
+
+<insert photo?> (not sure if possile)
+
+We want to take the time to thank each and everyone of you for joining us, for being
+excellent, and for contributing to this amazing community. We're inspired to see you
+all learning and contributing to open source during the sprint, and we
+hope you've been inspired to continue contributing. The Python community is better
+because of you 💜
+
+This is our first time organizing the mentored sprint, and numerous people
+have asked when will we do it again! Before we can get to that, we'd like to hear
+about your experience as a new contributor at the sprint.
+
+We are including an anonymous survey to hear your thoughts on what we can do even better next time. 
+
+Give Feedback: http://bit.ly/contributors-feedback-mentoredsprints
+
+# Thanks to mentors
+
+We're thankful to our mentors for volunteering their time and sharing their knowledge
+and passion with all of us. Please take a moment to give them a shout out and celebrate
+success using the [#mentoredsprint #pycon2019](https://twitter.com/search?q=%23mentoredsprint%20%23pycon2019&src=typd) hashtag on Twitter.
+
+# Thanks to y'all
+
+The mentored sprint for diverse beginners was a great event only because you showed
+up and were the best version of yourself. Thanks for contributing to the community!
+
+# Thanks to PyCon staff and The PSF
+
+We want to extend another big thank you to PyCon Hatchery committee, PyCon staff members
+and The PSF for their support in making the Mentored Sprint for Diverse Beginners a success.
+
+From Tania, Nina, Nikoleta & Mariatta
```

Comment:
Should we include a way to keep this community active before the next event?

Mailing list? Newsletter for news and updates? Slack channel somewhere (not a new slack)?